examples/asyncio/main.py

```diff
@@ -78,6 +79,7 @@ async def client_produce(client, data_items, queue_a, queue_b, n_clients):
 
 
 async def main():
+    Prio_init()
```

Comment:
kind of ashame we need to do this manually now (I assume there's a good reason)

Reply:
There is, it mostly deals with how NSS behaves under the hood. It's set up to be application global, and doesn't like it when the context is shutdown during the middle of execution. Hard to know when things get initialized during async execution (which is why I don't use the context manager here).